component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -158,7 +159,11 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
             throws AuthenticationFailedException {
 
         String username = null;
-
+        String showAuthFailureReasonOnLoginPage = null;
+        Map<String, String> parameterMap = getAuthenticatorConfig().getParameterMap();
+        if (parameterMap != null) {
+            showAuthFailureReasonOnLoginPage = parameterMap.get(TOTPAuthenticatorConstants.CONF_SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE);
```

Comment:
check the line length. if it is exceeding 120 characters, break this into 2 lines

Reply:
Addressed in 8fce69f